packages/eslint-config-4catalyzer/rules.js

```diff
@@ -27,8 +27,10 @@ module.exports = {
   }],
   'no-unused-vars': ['error', {
     vars: 'all',
+    varsIgnorePattern: '^_',
     args: 'after-used',
     ignoreRestSiblings: false,
+    argsIgnorePattern: '^_',
```

Comment:
do you want to insist on a trailing number? I guess it's not likely you'd be spreading out private fields with the underscore. 

idk, either is good! whatever you think 👍

Reply:
yeah i can't imagine spreading out private fields that way... if it happens we can fix